scripts/deploy.sh

```diff
@@ -197,7 +197,7 @@ else
     error $? "Could not pushd /data/socorro"
     PYTHONPATH=. ../socorro-virtualenv/bin/python \
         ../socorro-virtualenv/bin/alembic \
-        -c config/alembic.ini upgrade head &> /var/log/socorro/alembic.log
+        -c config/alembic.ini upgrade heads &> /var/log/socorro/alembic.log
```

Comment:
Would it be useful to see the log during the procedure?  If so, I would prefer `tee` to redirecting STDOUT, especially since this won't send STDERR to said log in the case of problems.  Possibly something like this instead?
```bash
-c config/alembic.ini upgrade heads 2>&1 | tee /var/log/socorro/alembic.log
```